src/backend/aspen/api/views/tests/test_create_phylo_runs.py

```diff
@@ -51,6 +53,49 @@ async def test_create_phylo_run(
     assert response["group"]["name"] == group.name
     assert response["user"]["name"] == user.name
     assert response["user"]["id"] == user.id
+    assert response["pathogen"]["slug"] == pathogen.slug
+    assert "id" in response
+
+
+async def test_create_phylo_run_mpx(
```

Comment:
Make sure we can create a phylo run with a non-default pathogen and everything works.